ovos_core/intent_services/commonqa_service.py

```diff
@@ -104,135 +121,161 @@ def match(self, utterances, lang, message):
                 message.data["utterance"] = utterance
                 answered = self.handle_question(message)
                 if answered:
-                    match = ovos_core.intent_services.IntentMatch('CommonQuery', None, {}, None, utterance)
+                    match = ovos_core.intent_services.IntentMatch('CommonQuery',
+                                                                  None, {},
+                                                                  None,
+                                                                  utterance)
                 break
         return match
 
-    def handle_question(self, message):
-        """ Send the phrase to the CommonQuerySkills and prepare for handling
-            the replies.
+    def handle_question(self, message: Message):
+        """
+        Send the phrase to CommonQuerySkills and prepare for handling replies.
         """
-        self.searching.set()
-        self.waiting = True
-        self.answered = False
         utt = message.data.get('utterance')
+        sid = SessionManager.get(message).session_id
+        # TODO: Why are defaults not creating new objects on init?
+        query = Query(session_id=sid, query=utt, replies=[], extensions=[],
+                      query_time=time.time(), timeout_time=time.time() + 1,
+                      responses_gathered=Event(), completed=Event(),
+                      answered=False)
+        assert query.responses_gathered.is_set() is False
+        assert query.completed.is_set() is False
```

Comment:
can this give a LOG about what went wrong and how to fix it instead of throwing a bare exception?